shm/store.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "cargo.hpp"
+#include "item.hpp"
+#include "player.hpp"
+#include "timeSHM.hpp"
+
+constexpr size_t numerOfItems{3};
+
+enum class Response { done, lack_of_money, lack_of_cargo, lack_of_space };
+
+class Store {
+public:
+    Store(Time* gameTime);
+    Response buy(Alcohol* alco, size_t amount, Player* player);
+    Response buy(Fruit* fruit, size_t amount, Player* player);
+    Response buy(Item* item, size_t amount, Player* player);
+    Response sell(Alcohol* alco, size_t amount, Player* player);
+    Response sell(Fruit* fruit, size_t amount, Player* player);
+    Response sell(Item* item, size_t amount, Player* player);
```

Comment:
Only one function for sell and for buy. Here I see that buy and sell is not static method. Is this code compile :)? And not crash?